lua/rustaceanvim/config/init.lua

```diff
@@ -117,6 +118,22 @@ vim.g.rustaceanvim = vim.g.rustaceanvim
 ---@alias dap_adapter_type_executable "executable"
 ---@alias dap_adapter_type_server "server"
 
+---@class DapClientConfig
+---@field type string The dap adapter to use
+---@field name string
+---@field request dap_config_request_launch | dap_config_request_attach | dap_config_request_custom The type of dap session
+---@field cwd? string Current working directory
+---@field program? string Path to executable for most DAP clients
+---@field args? string[] Optional args to DAP client, not valid for all client types
+---@field env? string  Environmental variables
+---@field initCommands? string Initial commands to run, `lldb` clients only
```

Comment:
question: Is this `string` or `string[]`?

Reply:
It's a string[]

https://github.com/vadimcn/codelldb/blob/master/MANUAL.md#starting-a-new-debug-session

Reply:
```suggestion
---@field initCommands? string[] Initial commands to run, `lldb` clients only
```